services/traction/requirements.txt

```diff
@@ -1,3 +1,33 @@
-uvicorn[standard]
-gunicorn
-fastapi
\ No newline at end of file
+alembic==1.7.5
```

Comment:
this is just the output of `pip freeze` from within the container, might be more flexible ways of doing this.